src/dimwits/quantity/test/operations.test.cpp

```diff
@@ -114,6 +114,44 @@ TEST_CASE("operations"){
       REQUIRE( generate_unit == Centi<Meter>{} );
     }
   }
+
+  SECTION("times equals"){
+    SECTION("scalar"){
+      quantity::Type< Centi<Meter> > length = 1.0 * meter;
+      length *= 5.0;
+      REQUIRE( 500.0 == length.value );
+
+      auto generate_unit = decltype(length)::Units();
+      REQUIRE( generate_unit == Centi<Meter>{} );
+    }
+    SECTION("unitless"){
+      quantity::Type< Centi<Meter> > length = 1.0 * meter;
+      length *= 5.0 * meter / meter;
+      REQUIRE( 500.0 == length.value );
```

Comment:
This looks like it should fail, until you realize that it is centimeters and not meters.